tee-worker/omni-executor/rpc-server/src/methods/pumpx/export_wallet.rs

```diff
@@ -60,6 +60,24 @@ pub fn register_export_wallet(module: &mut RpcModule<RpcContext>) {
 				ErrorObject::owned::<()>(AES_KEY_CONVERT_FAILED_CODE, "AesKey convert failed", None)
 			})?;
 
+			// verify user_id and user_email matches
+			let Ok(res) = ctx.pumpx_api.get_account_user_id(params.user_email.clone()).await else {
+				log::error!("Failed to call get_account_user_id");
+				return Err(
+					ErrorCode::ServerError(PUMPX_API_GET_ACCOUNT_USER_ID_FAILED_CODE).into()
+				);
+			};
+
+			if res.user_id != params.user_id {
```

Comment:
If we query it from backend anyway can we simply not accept it from frontend ?

Reply:
I think it gives us extra chance to tell F/E something goes wrong early.
If F/E doesn't pass in `user_email` - it could still give the wrong/arbitrary `user_id`. Now at least they need to pass in matching pairs.